OMCompiler/SimulationRuntime/c/simulation/solver/solver_main.c

```diff
@@ -700,9 +700,9 @@ int solver_main(DATA* data, threadData_t *threadData, const char* init_initMetho
   simInfo->useStopTime = 1;
 
   /* if the given step size is too small redefine it */
-  if ((simInfo->stepSize < MINIMAL_STEP_SIZE) && (simInfo->stopTime > 0)){
```

Comment:
What's  the reason for `simInfo->stopTime > 0`? So if we would simulate on the Intervall `[-2e-12, -1e-12]` the integrator could set the stepSize under some minimal step size.

Reply:
I guess it's just a clumsy way to prevent failing tests if simInfo->numSteps turns to 0.

Reply:
@AnHeuermann Thanks for the catch. I have changed it with the assumption that we always simulate forward in time.